### PR TITLE
Submodule update

### PR DESCRIPTION
**Pull request by sbesson**
- update `.gitmodules` to point at the `ome` org following migration
- bump all submodules to the HEAD of the `master` branch (`git submodule update --remote`)
- update Travis CI infrastructure to handle roles using either `ome-ansible-molecule` or `ome-ansible-molecule-dependencies`
- update `README` and add `LICENSE.md`

### CODE REVIEW

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Copyright (c) 2019, University of Dundee
```
Comment (manics) [inline LICENSE.md]: Shouldn't the name of the licence be included?

Comment (sbesson) [reply LICENSE.md]: I don't think so. At least neither https://opensource.org/licenses/BSD-2-Clause nor https://github.com/licensee/licensee (which is the library used by GitHub for detection) include it. Do you have an example where the name is included?

Comment (manics) [reply LICENSE.md]: I don't, but just looking at the licence I couldn't tell whether it was a standard opensource licence, or a custom one which needs to be read carefully.

Comment (sbesson) [reply LICENSE.md]: Understood. For reference, I simply copied/pasted a BSD-2 license from one of our repos and updated the header line. My assumption is that any custom changes to the license text will be flagged by GitHub not associating BSD-2 to the repository on merging.

Comment (sbesson) [reply LICENSE.md]: Update: to quickly test the above, I fast-forwarded the `master` branch of my fork to the HEAD of this PR and https://github.com/sbesson/ansible-roles correctly detects the repo as `BSD-2`

**Comment by sbesson** [comment]
Merging so that the weekly cron job starts testing the latest version of these roles. I'll also set up a job regularly updating the submodules (like other super-repos)

**Pull request merged**